EIPS/eip-6269.md

```diff
@@ -0,0 +1,69 @@
+---
+eip: 6269
+title: Full EVM Equivalence
+description: Canonicalise the definition of Full EVM Equivalence
+author: Pascal Caversaccio (@pcaversaccio)
+discussions-to: https://ethereum-magicians.org/t/evm-equivalence-eip-6269-and-ethereum-stack-compatibility-definition/10044
+status: Draft
+type: Meta
```

Comment:
Should be type informational

Reply:
My original draft was of type _informational_, @Pandapip1 suggested the _meta_ type https://github.com/ethereum/EIPs/pull/6269#discussion_r1063634390

Reply:
Yeah I think that is the correct type. Meta is used for processes around Ethereum which require consensus like the EIP process, hard forks, testnets, etc. This EIP is unenforceable (but still useful from an informational standpoint) and so I think this is the correct type.

Reply:
I think the question is about whether it can be ignored or not - imho it should not be ignored (otherwise my initiative is somehow useless). And if I understand correctly, an _informational_ EIP can be ignored whilst a _meta_ EIP cannot be ignored. Not sure what's the approach here to find a consensus but I want to clearly define a very important semantics that cannot be simply ignored by the community...

Reply:
> Yeah I think that is the correct type. Meta is used for processes around Ethereum which require consensus like the EIP process, hard forks, testnets, etc. This EIP is unenforceable (but still useful from an informational standpoint) and so I think this is the correct type.

It's not enforceable, but it's nonetheless not an EIP that can be ignored. Defining something, IMO, is Meta, while describing an aspect of something is Informational.

Reply:
Please review the EIPs that are classified as "Meta" and then review the EIPs that are classified as "Informational". This EIP is an Informational EIP because it does not affect anything related to Ethereum "process" (again, see definition of meta type in EIP-1). It is providing a definition for a term to the community, not proposing any new feature or process change, and is advisable to follow, but is not enforceable.